.github/workflows/docker-build-push.yml

```diff
@@ -68,19 +68,26 @@ jobs:
       # download data
       - name: Download data
         run: bash scripts/download-data.sh
-      
+
       - name: Render all pages to html
         run: |
           docker run \
           --mount type=bind,target=/home/rstudio,source=$PWD \
           ccdl/refinebio-examples \
           snakemake --cores 2 --forceall
 
-      - name: Commit changed html
+      # If we are on the staging branch, do not publish to github pages
+      - name: Commit changed html back to staging branch
+        if: github.ref == 'refs/heads/staging'
         run: |
           git add -A
-          git commit -m 'Render html' || echo "No changes to commit"
-          git push origin gh-pages || echo "No changes to push"
-
-
+          git commit -m 'Render html, do not publish' || echo "No changes to commit"
+          git push origin staging || echo "No changes to push"
```

Comment:
If the goal here is to commit back to staging, we will need to change the checkout before committing. The way this is currently set, it will make the changes in `gh-pages`, but push `staging`, which will be unchanged. 

I have not yet looked at the settings for `staging`, but if we are to follow the settings for `master`, then that branch should be protected as well. In that case the bot will not be able to push to it because pushes can only come from approved PRs. The easiest way around this would be to create another branch, `gh-pages-stages` if you will, that we could then push to from the staging branch.

Reply:
I've set `staging` to have the same settings that `master` does. 

`gh-pages-stages` feels a tad janky, but I guess its fine?

Reply:
it was a kind of a joke for a rhyme, but if you start the branch name with `gh-pages`, it will use the same settings. I don't know that it needs to be protected at all though. Just don't include `staging` in the name or it will inherit the restrictions from `staging`

Reply:
I didn't put protections on `gh-pages-stages` btw.

Reply:
It will inherit them because it has `gh-pages` in the name. This should not matter though given the current setup.

Reply:
OH right.